packages/bpk-component-image/src/BpkImage-test.js

```diff
@@ -68,8 +74,10 @@ describe('BpkImage', () => {
         src="./path/to/image.jpg"
       />,
     ).toJSON();
+
```

Comment:
Where these changes actually related to an upgrade or something you just did to make Arrange, Act, Assert more clear?

Reply:
I just added new lines between tests and statements tbh. yet another call for prettier

Reply:
👍